presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -78,6 +79,24 @@ public void testViews()
         execute("DROP VIEW IF EXISTS tpch.test_view");
     }
 
+    @Test
+    public void testSelectNativeDataTypeColumn()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.test_other_data_type (_foo UUID)");
+        assertTrue(queryRunner.tableExists(getSession(), "test_other_data_type"));
+
+        execute("INSERT INTO tpch.test_other_data_type VALUES ('00000000-0000-0000-0000-000000000000')");
+        assertEquals(
```

Comment:
Maybe consider using `assertQuery` to ensure not only row count but also type match and returned result?